schemas/logs/ossec/event_info.yml

```diff
@@ -23,11 +23,11 @@ fields:
       - name: level
         required: true
         description: The level of the rule (0 to 16). Alerts and responses use this value.
-        type: int
+        type: bigint
```

Comment:
Judging by [this](https://github.com/panther-labs/panther-enterprise/blob/master/internal/log_analysis/log_processor/parsers/osseclogs/eventinfo.go),  fields that were listed as `int` in Go, are now marked as `bigint` here. 

Looks good change. How did you generate it?